messaging/kafka/plugin_impl_kafka.go

```diff
@@ -60,6 +65,10 @@ func FromExistingMux(mux *mux.Multiplexer) *Plugin {
 
 // Init is called at plugin initialization.
 func (plugin *Plugin) Init() (err error) {
+	return plugin.initOnce.Do(plugin.init)
+}
+
+func (plugin *Plugin) init() (err error) {
```

Comment:
this method is always called once (now I mean only from one place), so I would put is as an anonymous function to Init(). Also applies for AfterInit, Close and same functions in other patches

Reply:
@VladoLavor I can see your point of view.  The reason I did it this way is that it significantly improves readability and reduces probabilities of mistakes due to closure with an anonymous function.